src/config.js

```diff
@@ -0,0 +1,1603 @@
+/** @babel */
+
+/*
+ * decaffeinate suggestions:
+ * DS101: Remove unnecessary use of Array.from
```

Comment:
Removed some instances of Array.from but mostly kept them in to emulate the behaviour the Coffeescript code.